client/client.go

```diff
@@ -1397,7 +1397,11 @@ func (c *Client) collectHostStats() {
 			c.resourceUsageLock.Lock()
 			c.resourceUsage = ru
 			c.resourceUsageLock.Unlock()
-			c.emitStats(ru)
+
+			// Publish Node metrics if operator has opted in
+			if c.config.PublishNodeMetrics {
```

Comment:
@slackpad We do that in the `emitMetrics` method of task runners, this is for the `emitMetrics` method of the Client which sends host related metrics.